Web/Scotty/Action.hs

```diff
@@ -65,6 +66,7 @@ import           Network.Wai
 
 import           Numeric.Natural
 
+import qualified Web.FormUrlEncoded         as F
```

Comment:
I've just noticed that CI fails because this module is not in scope. @mitchellwrosen could you add the 'http-api-data' import in the cabal file?